common/http/src/main/java/io/helidon/common/http/Utils.java

```diff
@@ -80,6 +80,22 @@ private Utils() {
         return result;
     }
 
+    /**
+     * Unwrap from double-quotes - if exists.
+     *
+     * @param str string to unwrap.
+     * @return unwrapped string.
+     */
+    public static String unwrap(String str) {
```

Comment:
Our APIs should neither expect nor return `null`. Please refactor the method and usage to only invoke if not null

Reply:
I've moved that method from an existing place, I take the blame since I'm making it "public" ; will make the change.